<launch>

  <!-- start Webots -->
  <arg name="no-gui" default="false," doc="Start Webots with minimal GUI"/>
  <include file="$(find hunter_webots_sim)/launch/webots.launch">
    <arg name="mode" value="realtime"/>
    <arg name="no-gui" value="$(arg no-gui)"/>
    <arg name="world" value="$(find hunter_webots_sim)/simulator/worlds/hunter_indoor.wbt"/>
  </include>

  <arg name="auto-close" default="false" doc="Startup mode"/>
  <node name="hunter_webots_node" pkg="hunter_webots_sim" type="hunter_webots_node" required="$(arg auto-close)" output="screen">
    <param name="use_sim_time" value="true" />
    <param name="clock" value="/clock" />
    <param name="odom_frame" type="string" value="odom" />
    <param name="base_frame" type="string" value="base_link" />
    <param name="simulated_robot" type="bool" value="true" />
    <param name="sim_control_rate" value="50" />
  </node>

  <include file="$(find hunter_description)/launch/load_nav_model.launch"/>

</launch>
